config/FCChh_DelphesCard_WithDipole_v00.tcl

```diff
@@ -37,7 +37,7 @@ TowerMerger
 EFlowMerger
 
 PhotonEfficiency
-PhotonEnergySmearing
+#PhotonEnergySmearing

```

Comment:
Delphes does not like comments in the ExecutionPath.
to make it usable one needs to remove '#PhotonEnergySmearing'